docs/index.md

```diff
@@ -50,155 +55,91 @@ Compose has commands for managing the whole lifecycle of your application:
 ## Compose documentation
 
 - [Installing Compose](install.md)
+- [Getting Started](gettingstarted.md)
 - [Get started with Django](django.md)
 - [Get started with Rails](rails.md)
 - [Get started with WordPress](wordpress.md)
 - [Command line reference](./reference/index.md)
 - [Compose file reference](compose-file.md)
 
-## Quick start
-
-Let's get started with a walkthrough of getting a simple Python web app running
-on Compose. It assumes a little knowledge of Python, but the concepts
-demonstrated here should be understandable even if you're not familiar with
-Python.
-
-### Installation and set-up
-
-First, [install Docker and Compose](install.md).
-
-Next, you'll want to make a directory for the project:
-
-    $ mkdir composetest
-    $ cd composetest
-
-Inside this directory, create `app.py`, a simple Python web app that uses the Flask
-framework and increments a value in Redis. Don't worry if you don't have Redis installed, docker is going to take care of that for you when we [define services](#define-services):
-
-    from flask import Flask
-    from redis import Redis
+## Features
```

Comment:
This is what it is called a stacked head --- two heads in a row with no intervening text.  I'd recommend you put links to the features so users can jump or just see them in a row.  Also, make the head reflect the main thought of each section

---->

The features of Compose that make it effective are:

- [Volume preservation](#volume-preservation)
- [Configuration caching](#configuration-caching)
- [Variables and extending](#variables-and-extending)